Tribler/Core/torrentstore.py

```diff
@@ -129,6 +137,15 @@ def close(self):
         self.flush()
         self._db = None
 
+    def _get_write_batch(self):
```

Comment:
Almost there! make this a global method that gets declared in either sides of the try/except so the class doesn't need to know about different leveldb implementations.